hyperSpec/vignettes/fileio.Rmd

```diff
@@ -1215,10 +1215,10 @@ more details for an already existing import filter.
 \index{PerkinElmer!Fluorescence}
 \index{PerkinElmer!ASCII long}
 
-The raw spectra of the `flu`{.r} data set (see also the respective vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
+The raw spectra of the `flu`{.r} data set (see also the separate vignette) are in PerkinElmer's ASCII file format, one spectrum per file.
```

Comment:
Add the link to the vignette (in the same manner Bryan introduced in `hyperSpec.Rmd`)